.misc/generate_dictionary.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# generate_dictionary.py
+#
+# Generates custom documentation dictionary for SpellCheckBear
+
+import os
+import appdirs
+import sys
+import shutil
+from scspell import __file__ as scfile
+
+
+class GenerateDictAction:
+    '''Represents the action of generating a new dictionary based on a
+    custom dictionary.'''
+    if sys.platform == 'win32':
+        USER_DATA_DIR = appdirs.user_data_dir(
+            appname='scspell', appauthor=False, roaming=True)
+    else:
+        USER_DATA_DIR = os.path.join(os.path.expanduser('~'), '.scspell')
```

Comment:
you should be able to use appdirs for non-windows also.

Reply:
I tried but `.scspell` pulls from specific location. (CI builds fail), it needs to have this distinction for *nix systems